model/sbvr/supervisor.sbvr

```diff
@@ -0,0 +1,298 @@
+Vocabulary: supervisor
+
+-- "(Type)" refers to the target database type to be autogenerated
+
+-- GENERIC TERMS
+
+Term: name
+	Concept Type: Short Text (Type)
+
+Term: status
+	Concept Type: Short Text (Type)
+
+Term: uuid
+	Concept Type: Text (Type)
+
+
+
+-- DEVICE
+
+Term: device type
+	Concept Type: Short Text (Type)
+
+Term: ip address
+	Concept Type: Short Text (Type)
+
+Term: mac address
+	Concept Type: Short Text (Type)
+
+Term: hostname
+	Concept Type: Short Text (Type)
+	-- This doesn't need to be a database type (the SV calls the hostname service under the hood) but not sure how to express that
+
+Term: device
+
+	Fact type: device is of device type
+		Synonymous Form: device type describes device
+		Necessity: each device is of exactly one device type.
+
+	Fact type: device has ip address
+		Necessity: each device has at least one ip address.
+
+	Fact type: device has mac address
+		Necessity: each device has at least one mac address.
+
+	Fact type: device has status
+		Necessity: each device has exactly one status.
+		Definition: "Provisioning" or "Post-Provisioning" or "Stopping" or "Starting" or "Downloading" or "Installing" or "Idle" 
+		-- TODO: these are partially taken from v2 API reference which may not be ideal. Decide on status strings for v3.
+
+	Fact type: device has name
+		Necessity: each device has exactly one name.
+
+	Fact type: device has uuid
+		Necessity: each device has exactly one uuid.
+		Necessity: each uuid is of exactly one device.
+		Necessity: each device has a uuid that has a Length (Type) that is equal to 32.
+
+	Fact type: device has hostname
+		Necessity: each device has exactly one hostname.
+
+	Fact type: device is connected to vpn
+
+
+
+-- DEVICE METRICS
+
+Term: cpu id
```

Comment:
This might make more sense on the device model rather than the device metrics model as it is a constant.